workflowdata.json

```
@@ -0,0 +1,492 @@
+{
+  "data": [
```

Not sure there's any reason for this "data" key. How about removing this outer layer? i.e. basically replace this:

```
{"data": [<actual contents>]}
```

with just this:

```
[<actual contents>]
```

```suggestion
[
```